keyboards/kisakeyluxury/qtz/keymaps/default/config.h

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2023 kisakeyluxury
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#ifdef COMBO_ENABLE
+#    define COMBO_TERM 200
+#endif
```

Comment:
This can be moved to combo.term in the info.json

Reply:
no disrespect @drashna, but the combo feature, like encoder_map and various term tapping settings, feels best placed as a keymap config/setting rather than at keyboard.